logging/bin/getlogs.py

```diff
@@ -47,39 +52,49 @@ def validate_input(dict):
     if(len(sys.argv) == 1): ##Check if any args have been provided
```

Comment:
Doesn't need to happen right now but I think when I moved this code over from gitlab and updated some things that we no longer need this check, I'm pretty sure args are already grabbed by the time we've made it this far in the code. Def could be wrong here though and this isn't something that needs to happen now, but we should likely make note of this and clean it up if so.